apps/meteor/ee/client/hooks/useTagsList.ts

```diff
@@ -12,17 +11,19 @@ type TagsListOptions = {
 	viewAll?: boolean;
 };
 
+type TagsItemsList = { _id: string; label: string; value: string; _updatedAt: Date };
```

Comment:
This name in my opinion does not correspond to the type. The type seem to be of a singular item, so something like `TagsItem` _(or similar)_ might be more appropriate.